traits/stubs_tests/examples/completeness.py

```diff
@@ -21,6 +21,6 @@ class HasName(HasStrictTraits):
 
 def try_assigning_age(x: HasName, new_name: str):
     try:
-        x.age = new_name
+        x.name = new_name
```

Comment:
This was my code; I have _no_ idea what I was thinking.

Reply:
and we will never know. until Enthought succeeds in creating the time machine that is.